### PR TITLE
CI: seed container image builds, trigger image sync to test pulp

### PR DESCRIPTION
**Pull request by markgoddard**

### CODE REVIEW

File: .github/workflows/container-image-build.yml [hunk 1]
```diff
@@ -77,12 +88,71 @@ jobs:
           kayobe overcloud container image build ${{ github.events.input.regexes }} --push
         env:
           KAYOBE_VAULT_PASSWORD: ${{ secrets.KAYOBE_VAULT_PASSWORD }}
+        if: github.events.input.overcloud
+
+      - name: Display the overcloud container image tag
+        run: |
+          echo "$(cat ~/kolla_tag)"
+        if: github.events.input.overcloud
```
Comment (priteau) [inline .github/workflows/container-image-build.yml]: I think all occurrences of `github.events` should be `github.event`.

However I haven't yet verified that the change for `input.regexes` above fixed it.

**Review by priteau — CHANGES_REQUESTED** [review]

File: .github/workflows/container-image-build.yml (hunk 1 above)
Comment (markgoddard) [reply .github/workflows/container-image-build.yml]: Good catch

Comment (priteau) [reply .github/workflows/container-image-build.yml]: And it should actually be `github.event.inputs` rather than `github.event.input` 😓

**Review by priteau — APPROVED** [review]

**Comment by priteau** [comment]
@markgoddard Please check that you are happy with the additional commit I pushed.

File: .github/workflows/container-image-build.yml (hunk 1 above)
Comment (markgoddard) [reply .github/workflows/container-image-build.yml]: > @markgoddard Please check that you are happy with the additional commit I pushed.

Thank you for fixing up. I spotted this in another workflow :( Would be better if github failed on these invalid references.

**Pull request merged**